src/Parse/WordParser.py

```diff
@@ -9,6 +9,12 @@ def __init__(self, blob: bytearray):
         self.doc = Document(self.docx_bytes)
         return
     
+    def ensure_https(self, url):
+        # regex checks if URL starts with http:// or https://
+        if not re.match(r'^https?:\/\/', url):
+            url = 'https://' + url
+            return url
+        
```

Comment:
What happens if the regex matches? Think we need to return the original `url` instead of `None`

Reply:
Otherwise loosk good!